src/app/session/session.service.ts

```diff
@@ -149,6 +150,11 @@ export class SessionService {
   public logout(next: string = null): void {
     localStorage.removeItem(AUTHORIZATION_TOKEN);
     localStorage.removeItem(REFRESH_TOKEN);
+    this.store.dispatch(new sharedActions.FetchMaps());
+    this.store.dispatch(new sharedActions.FetchModels());
+    this.store.dispatch(new sharedActions.FetchProjects());
+    this.store.dispatch(new sharedActions.FetchJobs());
+    this.store.dispatch(new sharedActions.FetchDesigns());
```

Comment:
So this will update the store with only public data? I have a couple of concerns:

- If adding new data types in the store we always need to remember to dispatch the action here. maybe it's better with a single action to update all data?
- If there is a network delay or the request for some reason doesn't finish, stale data would be available in the store. Maybe the store could be emptied first just to be sure.